interface/usergroup/user_admin.php

```diff
@@ -466,6 +343,36 @@ function authorized_clicked() {
 <tr>
   <td><span class="text"><?php xl('Provider Type','e'); ?>: </span></td>
   <td><?php echo generate_select_list("physician_type", "physician_type", $iter['physician_type'],'',xl('Select Type'),'physician_type_class','','',''); ?></td>
+  <td>
+    <span class="text"><?php echo xlt('Main Menu Role'); ?>: </span>
+  </td>
+  <td>
+    <?php
+    $mainMenuRole = $iter["main_menu_role"];
+    ?>
+    <select name="main_menu_role">
+      <option value="standard" <?php echo ($mainMenuRole == "standard") ? "selected" : ""?>><?php echo xlt("Standard"); ?></option>
+      <option value="answering_service" <?php echo ($mainMenuRole == "answering_service") ? "selected" : ""?>><?php echo xlt("Answering Service"); ?></option>
+      <option value="front_office" <?php echo ($mainMenuRole == "front_office") ? "selected" : ""?>><?php echo xlt("Front Office"); ?></option>
+      <?php
+        $customMenuDir = $GLOBALS['OE_SITE_DIR'] . "/documents/custom_menus";
+        if (file_exists($customMenuDir)) {
+          $dHandle = opendir($customMenuDir);
+          while (false !== ($menuCustom = readdir($dHandle))) {
+            // Only process files that contain *.json
+            if (preg_match("/.json$/", $menuCustom)) {
+              $selectedTag = ($mainMenuRole == $menuCustom) ? "selected" : "";
+              echo "<option value='" . attr($menuCustom) . "' " . $selectedTag . ">";
+              // Drop the .json and translate the name
+              echo xlt(substr($menuCustom, 0, -5));
+              echo "</option>";
+            }
+          }
+          closedir($dHandle);
+        }
+      ?>
+    </select>
```

Comment:
@MatthewVita ,
I am planning to migrate above code (the creation of this select control) into services/UserService.php . I just wanted to make sure that would be in scope of the user service.